README.rdoc

```diff
@@ -0,0 +1,28 @@
+== README
+
+This README would normally document whatever steps are necessary to get the
```

Comment:
The ReadMe's a great place to add information about the project including how to get it running and where to find it on the web, I recommend deleting the Rails generated one and creating your own in markdown